rpcserver.go

```diff
@@ -5104,8 +5105,17 @@ func median(s []dcrutil.Amount) dcrutil.Amount {
 		return 0
 	}
 
+	sort.Sort(dcrutil.AmountSorter(s));
+
 	middle := len(s) / 2
-	return s[middle]
+
+	if len(s) == 0 {
+		return 0
+	} else if (len(s) % 2) != 0 {
+		return s[middle]
+	} else {
```

Comment:
Can remove the last else and just return. In fact, can probably be simplified as

 +	if len(s) == 0 {
+		return 0
+	if (len(s) % 2) != 0 {
+		return s[middle]
+	}
+	return (s[middle] + s[middle - 1]) / 2